include/cuco/detail/storage/aow_storage.cuh

```diff
@@ -113,6 +114,106 @@ class aow_storage_ref : public aow_storage_base<WindowSize, T, Extent> {
   {
   }
 
+  /**
+   * @brief Custom un-incrementable input iterator for the convenience of `find` operations.
+   *
+   * @note This iterator is for read only and NOT incrementable.
+   */
+  struct Iterator {
```

Comment:
```suggestion
  struct iterator {
```
I think that's the convention we use for nested classes.

Reply:
solved